### PR TITLE
Fix for #227 - smtp smtplib server_hostname

### PR DESCRIPTION
**Pull request by nicoh88**
"server_hostname cannot be an empty string or start with a leading dot"

### CODE REVIEW

File: icloudpd/email_notifications.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ def send_2sa_notification(
     from_addr = smtp_email if smtp_email else to_addr
     logger = setup_logger()
     logger.info("Sending 'two-step expired' notification via email...")
-    smtp = smtplib.SMTP()
+    #smtp = smtplib.SMTP()
+    smtp = smtplib.SMTP(smtp_host, smtp_port)
     smtp.set_debuglevel(0)
     smtp.connect(smtp_host, smtp_port)
```
Comment (menkej) [inline icloudpd/email_notifications.py:21]: This line is obsolete then

File: icloudpd/email_notifications.py [hunk 2]
```diff
@@ -15,7 +15,8 @@ def send_2sa_notification(
     from_addr = smtp_email if smtp_email else to_addr
     logger = setup_logger()
     logger.info("Sending 'two-step expired' notification via email...")
-    smtp = smtplib.SMTP()
+    #smtp = smtplib.SMTP()
```
Comment (menkej) [inline icloudpd/email_notifications.py:18]: commented line should be removed. Git keeps track... ;-)

**Review by menkej — CHANGES_REQUESTED** [review]
Please also add the fix for #227 to the CHANGELOG as well. Thanks!

**Comment by nicoh88** [comment]
Done 👍

File: icloudpd/email_notifications.py [hunk 3]
```diff
@@ -15,7 +15,7 @@ def send_2sa_notification(
     from_addr = smtp_email if smtp_email else to_addr
     logger = setup_logger()
     logger.info("Sending 'two-step expired' notification via email...")
-    smtp = smtplib.SMTP()
+    smtp = smtplib.SMTP(smtp_host, smtp_port)
     smtp.set_debuglevel(0)
     smtp.connect(smtp_host, smtp_port)
```
Comment (menkej) [inline icloudpd/email_notifications.py:20]: smtp.connect is implicit now. Explicit call should be removed.

**Review by menkej — CHANGES_REQUESTED** [review]
smtp.connect is implicit now. Explicit call should be removed.

File: icloudpd/email_notifications.py (hunk 3 above)
Comment (nicoh88) [reply icloudpd/email_notifications.py:20]: The best thing to do is to make the changes yourself. Close here.

**Comment by menkej** [comment]
Why did you delete this before merge?

**Comment by nicoh88** [comment]
> smtp.connect is implicit now. Explicit call should be removed.

I didn't know what else to do. So I thought you'd just change the two lines yourself before there were a few back and forth

**Comment by menkej** [comment]
Well, that's just normal review-process - I did not want to offend you or anything. I know it's just a two liner and the "overhead" of changelog and putting out a release is way more of work but that happens sometimes... If you'd just undelete the branch I could merge. Otherwise we just wasted time with this dialog and I do it myself (wich I had done already but could not test it). :-(

**Pull request closed without merging**